src/Symfony/Component/Routing/Generator/UrlGenerator.php

```diff
@@ -142,6 +142,17 @@ public function generate(string $name, array $parameters = [], int $referenceTyp
      */
     protected function doGenerate(array $variables, array $defaults, array $requirements, array $tokens, array $parameters, string $name, int $referenceType, array $hostTokens, array $requiredSchemes = []): string
     {
+        $queryParameters = [];
+
+        if (isset($parameters['_query'])) {
+            if (is_array($parameters['_query'])) {
```

Comment:
dunno why fabbot doesn't complain about this:
```suggestion
            if (\is_array($parameters['_query'])) {
```

Reply:
Done!